index.html

```diff
@@ -1,8 +1,153 @@
+      </figure>
+      </article>
+      
+      <center><br><a href = #top>Back to Top</a></center>
+      
+
+      <article id = "nugget">
+      <h2>Nugget</h2>
+      <p>In the winter of 2013, I was working at a retreat center in the Ozarks when the camp cat gave birth to yet another litter of kittens. This was a litter of three, all black jellybeans. Unfortunately, due to an extreme cold snap, only one kitten survived and her mother rejected her. I adopted her as my office mascot and named her Nugget. When I left the camp in 2014, Nugget came with me. Despite having been an indoor/outdoor cat for the first year and a half of her life, Nugget was more than content to become a fat, lazy house cat. Due to the trauma of her early kittenhood, Nugget is not easily trusting and has some anger issues. We've tried working them out but it usually ends in bloodshed. Approaching ten years old, Nugget is still content with being an only cat and lives with my parents where she enjoys yelling at them for food, snuggling with my dad, and tormenting my mom.</p>
+      <figure>
+         <img src = "images\NUGGET1.jpg" width="400" height = "300">
+         <figcaption>Nugget, being a goofball.</figcaption>
+      </figure> 
+         
+      <figure>
+         <img src = "images\NUGGET2.JPG" width="300" height="400">
+         <figcaption>Nugget, being a much larger goofball.</figcaption>
+      </figure>
+      </article>
+      
+      <center><br><a href = #top>Back to Top</a></center>
+      
+
+      <article id = "accio">
+      <h2>Accio</h2>
+      <p>Sometime around September 2012, I noticed a very malnourished calico cat hanging around my parents' house. (This was a trend throughout the years, people dumping animals.) I commented to my dad that I had noticed the cat, and he had, too. He told me very sternly to not feed it, and it would go away. Naturally, that night after he had gone to bed, I fed the cat a can of tuna. She was very appreciative and affectionate after she had her fill. I repeated this ritual for a few nights and then stealthily purchased some dry cat food which I would hide on the back deck for my new found friend. It didn't take long for my dad to notice that the cat wasn't leaving and no longer looked malnourished. He said I could keep feeding her but that he wasn't taking any responsibility for the cat. I named her Accio after the Harry Potter spell because I would stand on the back porch and shout, "Accio, kitty!" and she would come running. Despite my best efforts that winter, Accio refused to become an indoor cat and lived her life in the wild. She would get under the house and sleep on duct work to keep warm and had her fill of mice. Sadly, after a head injury and another attempt to keep her inside had failed, Accio did what wild animals do and found a quiet place to cross the rainbow bridge last year. She was, by far, the most affectionate cat I have ever had the pleasure of keeping.</p>
```

Comment:
When it comes to paragraphs like this, for readability, I recommend hitting enter after every sentence or so, so you don't have such a long single line where you need to scroll over so much. Don't worry, this won't change how it appears on your webpage, but just makes the code a little more readable!